extensions/geode-modules-session-internal/src/main/java/org/apache/geode/modules/session/internal/common/AbstractSessionCache.java

```diff
@@ -31,19 +31,20 @@
    * The sessionRegion is the <code>Region</code> that actually stores and replicates the
    * <code>Session</code>s.
    */
-  protected Region<String, HttpSession> sessionRegion;
+  Region<String, HttpSession> sessionRegion;
 
   /**
    * The operatingRegion is the <code>Region</code> used to do HTTP operations. if local cache is
    * enabled, then this will be the local <code>Region</code>; otherwise, it will be the session
    * <code>Region</code>.
    */
-  protected Region<String, HttpSession> operatingRegion;
+  Region<String, HttpSession> operatingRegion;
 
-  protected Map<CacheProperty, Object> properties =
-      new TypeAwareMap<CacheProperty, Object>(CacheProperty.class);
+  @SuppressWarnings("unchecked")
+  protected Map<CacheProperty, Object> properties = new TypeAwareMap<>(CacheProperty.class);
```

Comment:
I think the `unchecked` warning is because the underlying implementation of TypeAwareMap extends from a non-generic HashMap.  Not sure if that is something we can fix here or not.

Reply:
Yeah, thanks, I don't think we can do anything about this one here.